Amplify/Core/Support/Task+Seconds.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-extension Task where Success == Never, Failure == Never {
+public extension Task where Success == Never, Failure == Never {
```

Comment:
should this be public given that we don't own this type?

Reply:
https://github.com/aws-amplify/amplify-ios/issues/2154